2023-07-26 00:28:07.867 UTC [920837] LOG:  starting PostgreSQL 15.3 on x86_64-pc-linux-gnu, compiled by gcc (Ubuntu 11.3.0-1ubuntu1~22.04.1) 11.3.0, 64-bit
2023-07-26 00:28:07.867 UTC [920837] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2023-07-26 00:28:07.882 UTC [920837] LOG:  listening on Unix socket "/tmp/.s.PGSQL.5432"
2023-07-26 00:28:07.898 UTC [920840] LOG:  database system was shut down at 2023-07-26 00:28:07 UTC
2023-07-26 00:28:07.909 UTC [920837] LOG:  database system is ready to accept connections
2023-07-26 00:28:07.910 UTC [920843] LOG:  pg_partman master background worker master process initialized with role postgres
2023-07-26 00:28:08.706 UTC [920865] ERROR:  could not create unique index "idx_badindex_n"
2023-07-26 00:28:08.706 UTC [920865] DETAIL:  Key (n)=(10) is duplicated.
2023-07-26 00:28:08.706 UTC [920865] STATEMENT:  CREATE UNIQUE INDEX CONCURRENTLY idx_badindex_n ON tbl_badindex (n);
2023-07-26 00:28:16.293 UTC [921038] ERROR:  permission denied for schema repack at character 8
2023-07-26 00:28:16.293 UTC [921038] STATEMENT:  select repack.version(), repack.version_sql()
2023-07-26 00:28:19.222 UTC [921127] ERROR:  table name not found for OID 1
2023-07-26 00:28:19.222 UTC [921127] STATEMENT:  SELECT repack.get_order_by('issue321_idx'::regclass::oid, 1);
2023-07-26 00:28:19.223 UTC [921127] ERROR:  cache lookup failed for index 1
2023-07-26 00:28:19.223 UTC [921127] STATEMENT:  SELECT repack.get_order_by(1, 1);
